<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>start_new_room</class>
 <widget class="QWidget" name="start_new_room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка зала</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="btn_GameSettings">
     <property name="text">
      <string>Настройки Игры</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btn_StartTheGame">
     <property name="text">
      <string>Открыть зал для входа игроков</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btn_DebugStart">
     <property name="text">
      <string>Начать игру (с настройками по умолчанию) - отладка</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="btn_ServerSettings">
     <property name="text">
      <string>Настройки сервера</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
